### PR TITLE
Adding support for suppressing initial watching message

### PR DESCRIPTION
**Pull request by lukosanthropos**

### CODE REVIEW

**Comment by lukosanthropos** [comment]
See https://github.com/DocnetUK/gae-matt-daemon/pull/2

**Comment by dave-docnet** [comment]
Salut,

Is this PR still required?

Cordialement,
Dave

File: src/Docnet/MATT.class.php [hunk 1]
```diff
@@ -181,6 +188,18 @@ public function sms($str_recipient)
     }
 
     /**
+     * Should we suppress the initial message telling us that the watch has been created
+     *
+     * @param $bol_suppress
+     * @return $this
+     */
+    public function suppress_watch_message($bol_suppress)
```
Comment (tomwalder) [inline src/Docnet/MATT.class.php]: Can this default to `true` please?

**Comment by tomwalder** [comment]
I've left a small comment, which once resolved, will mean we can merge

**Comment by lukosanthropos** [comment]
Updated PR based on comment/request @dave-docnet @tomwalder please review

**Pull request merged**